types/kiosk-browser.d.ts

```diff
@@ -0,0 +1,88 @@
+declare namespace KioskBrowser {
+  export interface BatteryInfo {
+    discharging: boolean
+    level: number // Number between 0–1
+  }
+
+  export interface PrinterInfo {
+    // Docs: http://electronjs.org/docs/api/structures/printer-info
+    description: string
+    isDefault: boolean
+    name: string
+    status: number
+    // Added via kiosk-browser
+    connected: boolean
+    options?: { [key: string]: string }
+  }
+
+  export interface Device {
+    locationId: number
+    vendorId: number
+    productId: number
+    deviceName: string
+    manufacturer: string
+    serialNumber: string
+    deviceAddress: number
+  }
+
+  export interface UsbDrive {
+    deviceName: string
+    mountPoint?: string
+  }
+
+  export interface FileWriter {
+    /**
+     * Writes a chunk to the file. May be called multiple times. Data will be
+     * written in the order of calls to `write`.
+     */
+    write(data: Buffer | Uint8Array | string): Promise<void>
+
+    /**
+     * Finishes writing to the file and closes it. Subsequent calls to `write`
+     * will fail. Resolves when the file is successfully closed.
+     */
+    end(): Promise<void>
+  }
+
+  export interface Kiosk {
+    print(): Promise<void>
+    getPrinterInfo(): Promise<PrinterInfo[]>
+
+    /**
+     * Prints the current page to PDF and resolves with the PDF file bytes.
+     */
+    printToPDF(): Promise<Uint8Array>
+
+    getBatteryInfo(): Promise<BatteryInfo>
+    devices: import('rxjs').Observable<Iterable<Device>>
+    quit(): void
+
+    /**
+     * Opens a Save Dialog to allow the user to choose a destination for a file.
+     * Once chosen, resolves with a handle to the file to write data to it.
+     */
+    saveAs(): Promise<FileWriter | undefined>
+
+    // USB sticks
+    getUsbDrives(): Promise<UsbDrive[]>
+    mountUsbDrive(device: string)
+    unmountUsbDrive(device: string)
+
+    // storage
+    storage: {
+      set<K extends keyof M>(key: string, value: M[K]) : Promise<void>
+      get<K extends keyof M>(key: string) : Promise<M[K] | undefined>
+      remove(key: string) : Promise<void>
+      clear() : Promise<void>
+    }
+  }
+}
+
+// Disable `no-var` because using `var` ensures `kiosk` is a property on
+// `globalThis`, which makes it available both as plain `kiosk` and as
+// `window.kiosk`. Using `const` or `let`, as eslint suggests, will not make it
+// available at all. An alternative would be to add `kiosk` as a property to
+// the `Window` interface, but then we couldn't refer to `kiosk` without
+// `window`.
+// eslint-disable-next-line no-var
```

Comment:
FWIW, I'd prefer that we always use `window.kiosk` for clarity and to avoid "magic".

Reply:
Yeah this is the kiosk types file from  election manager

Reply:
I think that's how we do it everywhere, at least the way my code uses it.

Reply:
Great. Then can we remove the above comment block?

Reply:
OK, so this is more complex than it looks, at least in that if we remove the variable declaration, tsc fails, and I am not 100% sure of the right way to declare that `window.kiosk` is good, but `globalThis.kiosk` is not. So, because we're doing this the same way in other repos, I'm going to keep it as is.